.github/styles/kong/Relativeurls.yml

```diff
@@ -0,0 +1,7 @@
+extends: existence
+message: "Use relative URLs for docs URLs for example: [Get Started](/gateway/latest/get-started/) instead of [Get Started](https://docs.konghq.com/gateway/3.3.x/get-started/). If a direct URL is necessary, use {{ site.links.web }} instead of `https://docs.konghq.com`."
```

Comment:
```suggestion
message: "Use relative URLs for docs URLs. For example: [Get Started](/gateway/latest/get-started/) instead of [Get Started](https://docs.konghq.com/gateway/3.3.x/get-started/). If a direct URL is necessary, use {{ site.links.web }} instead of `https://docs.konghq.com`."
```

Reply:
I wonder if we can shorten this? Eg For example, use /gateway/latest/get-started/ instead of https://docs.konghq.com/gateway/3.3.x/get-started/.